src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -828,6 +829,80 @@ public static BinaryOperandsKind GetBinaryOperandsKind(BinaryOperationKind kind)
         }
     }
 
+    public interface IConversion
```

Comment:
The argument about having a common treatment of conversions between IOperation and SemanticModel is compelling. I'll go look at the treatment in SemanticModel.